src/components/Member/CurrentMembers.vue

```diff
@@ -86,37 +91,17 @@ export default {
 
 	methods: {
 		/**
-		 *
 		 * @param {object} m1 First member
 		 * @param {string} m1.userId First member user ID
-		 * @param {string} m1.displayName First member display name
-		 * @param {number} m1.level First member level
-		 * @param {number} m1.userType First member user type
 		 * @param {object} m2 Second member
 		 * @param {string} m2.userId Second member user ID
-		 * @param {string} m2.displayName Second member display name
-		 * @param {number} m2.level Second member level
-		 * @param {number} m2.userType Second member user type
 		 */
-		sortMembers(m1, m2) {
+		sortCurrentUserFirst(m1, m2) {
 			if (m1.userId === this.currentUser) {
 				return -1
 			} else if (m2.userId === this.currentUser) {
 				return 1
 			}
```

Comment:
What if neither of the two is the currentUser?
Right now this will return nothing / undefined, right?
Is that what you want?

Reply:
Well, strictly speaking we could also `return 0`. But to my knowledge that's the default - not to sort the two. Given that I moved sorting by user type and level into `src/store/circles.js`, this sort functions sole left purpose is to put `currentUser` first.

Reply:
Ahh... now i get it. `currentMembers` is already passed in presorted.
Let's still return 0 to follow the requirements for the sort function as described here:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort#parameters

> The return value should be a number whose sign indicates the relative order of the two elements: negative if a is less than b, positive if a is greater than b, and zero if they are equal. NaN is treated as 0.

Reply:
Oh... and I know this is accademic and probably has no real effect :monocle_face: 
Another requirement for the function to return stable results is for it to be reflexive:
```js
compareFn(a, a) === 0.
```

Reply:
Agreed, thanks for being thorough here :heart: Fixed now.